src/Numerics/DGmethods/DGmodel_kernels.jl

```diff
@@ -1451,7 +1451,7 @@ end
                 reverse_integral_load_aux!(
                     bl,
                     Vars{vars_reverse_integrals(bl, FT)}(l_V),
-                    Vars{vars_state(bl, FT)}(view(state, ijk, :, et)),
+                    Vars{vars_state(bl, FT)}(view(state, ijk, :, e)),
```

Comment:
Good catch!

Reply:
Thanks for checking this out , I was going to ping you to verify :D